WordPress/src/main/java/org/wordpress/android/ui/reader/repository/usecases/ShouldAutoUpdateTagUseCase.kt

```diff
@@ -0,0 +1,16 @@
+package org.wordpress.android.ui.reader.repository.usecases
+
+import kotlinx.coroutines.CoroutineDispatcher
+import org.wordpress.android.datasets.ReaderTagTable
+import org.wordpress.android.models.ReaderTag
+import org.wordpress.android.modules.BG_THREAD
+import javax.inject.Inject
+import javax.inject.Named
+
+class ShouldAutoUpdateTagUseCase @Inject constructor(
+    @Named(BG_THREAD) private val bgDispatcher: CoroutineDispatcher
+) : ReaderRepositoryDispatchingUseCase(bgDispatcher) {
+    fun get(readerTag: ReaderTag): Boolean {
```

Comment:
We might want to make this method suspendable and move it to a bg thread, wdyt?

Reply:
in commit 576c59f